frontend/tests/playwright/msw/handlers.ts

```diff
@@ -0,0 +1,35 @@
+import { RequestHandler, graphql } from 'msw';
+import { mockedProductQuery } from '@tests/jest/__mocks__/products';
+import { cloneDeep } from 'lodash';
+
+export const handlers: RequestHandler[] = [
+   graphql.query('findProduct', async (req, res, ctx) => {
+      if (
+         req.variables.handle ===
+         'iphone-6s-plus-replacement-battery-low-stocked'
+      ) {
+         return res(ctx.data(mockedProductQuery));
```

Comment:
Do we want to explicitly set the `quantityAvailable` in this if condition like we do for the out-of-stock option? (not a blocker)

Reply:
For consistency, I will make the change.

Reply:
Addressed in [5897df8](https://github.com/iFixit/react-commerce/pull/1216/commits/5897df83970a2810697a9f1e41c8c4988854918c)